doc/title.xml

```diff
@@ -0,0 +1,58 @@
+<!-- ------------------------------------------------------------------- -->
+<!--                                                                     -->
+<!--  title.xml               KBMag documentation                        -->
+<!--                                                                     -->
+<!--  Copyright (C) 1997-2017, Derek Holt                                --> 
+<!--                                                                     -->
+<!-- ------------------------------------------------------------------- -->
+
+<TitlePage>
+
+  <Title>&KBMAG;</Title>
+  <Subtitle>Knuth--Bendix on Monoids, and Automatic Groups</Subtitle> 
+  <Version>&KBMAGVERSION;</Version>   
+  <Date>&KBMAGRELEASEDATE;</Date> 
+  <Author> Derek Holt
+    <Address> 
+      Mathematics Institute<Br/> 
+      University of Warwick, Coventry, CV4 7AL 
+    </Address> 
+  </Author>
+
+<Abstract>
+The &KBMAG; package for &GAP;4 provides an interface to some `C' programs 
+for running the Knuth-Bendix completion program on finite semigroup, 
+monoid or group presentations, and for attempting to compute  automatic 
+structures of finitely presented groups. 
+<P/>
+The latest version is &KBMAGVERSION; of &KBMAGLONGRELEASEDATE;.  
```

Comment:
This sentence will be wrong most of the time, more accurate would be: "at the time this manual was created, the latest version was...". And as such, it is also pointless, because *that* version is already provided further above on the title page.

I.e.: just delete it?